pkg/downloader.go

```diff
@@ -148,15 +147,12 @@ func (d *Downloader) Run(ctx context.Context, dryRun, runOnce bool) error {
 
 // clean target working directory
 func (d *Downloader) clean(directory string) error {
-	cmd := exec.Command("rm", "-rf", directory)
-	cmd.Dir = d.workdir
-	err := cmd.Run()
+	err := os.RemoveAll(directory)
```

Comment:
iirc need to account for all the repos to clean being nested within the set `workdir`:

`os.RemoveAll(fmt.Sprintf("%s/%s", d.workdir, directory)`

Reply:
https://github.com/app-sre/git-partition-sync-consumer/pull/34/commits/70f66b7529c61af5c1ec83c9663c0f129695abea